### DISCUSSION TITLE
How do I permanently disable the new Dashboard?!

### DISCUSSION THREAD

**Discussion by Pingger** [netdata/netdata#15415] [Q&A]
Hi,

I just accessed one of my netdata nodes and was quite shocked, that the Dashboard became a Trashboard.
Takes about 1 Minute to load, to then be bloated with scripts, that cause the browser to lag.
In addition I couldn't find a thing!
I have several network interfaces, each for a different purpose, which separate limits, alarms and so on. 
In addition that Node also acts as a replication receiver of several other nodes.
What I absolutely do not need as a unified graph where all the interfaces are summed! Changing that to 'Average' 'Min' or 'Max' did not change a thing in the graph.
Trying to select a singular interface showed even worse behavior. Most of my Nodes have 'ens3' as their primary network interface. Those are all merged together into a single one and I was not able to split them apart in the new trashboard.
Another huge issue is that graphs that are supposed to be viewed side-by-side can't be viewed in such a way at all anymore. In this case: fping! I'm monitoring the ping times between my nodes and a few public servers. Again: What I don't need AT ALL is the sum of the latencies. I also don't need an average, which doesn't even work as before!
Using the "local copy" of the trashboard fixes the not working selectors for the graphs at least. But the really doesn't make the new trashboard any more usable.
Not even regarding the spiking cpu times of the netdata process and my browser when using the new trashboard.

So how do I disable the new trashboard entirely, so it stops being a nuisance?

I can't even go into /usr/share/netdata/ and replace all the index.html files so they imediatly redirect to the v1 version. But hey on a positive note: I can make the v1 interface which is also more flawed than the v0 interface redirect there, which I didn't know before.

and having access to the v0 interface actually made one of my nodes reappear and their alarms work again! It went dark with the update to the v1 interface and after spending days reinstalling netdata on both nodes I still couldn't get it to reappear, but hey I found something good, because the v2 UI is so bad

**Comment by ktsaou** [comment]
Hi @Pingger !

Thank you for your message!

From your message I understand that the main problem is that you need to have the network interfaces expanded. So, instead of having multiple interfaces in one chart, you need one network interface per chart. Correct me if I got it wrong.

The decision to merge charts was in way depicted by the community. We have users that have thousands of interfaces, disks, containers, even database tables and the problem with the old dashboard is that does not scale well. If you add a few thousand charts to it, everything becomes unmanageable. So we needed to solve this problem somehow.

But yes, I agree that the expanded view is needed! We have planned an expanded view of the new charts, that you will be able to drill down by instance (similar to the old dashboard), even by dimension (which is needed if you need to identify anomalies).

I will meet with our front-end team tomorrow morning and discuss this issue. We have a few blockers that we need to address immediately, but I think the first feature to implement immediately after, is the ability to drill down per instance.

You make a few more comments that I need to understand:

### Time to load the dashboard

> Takes about 1 Minute to load, to then be bloated with scripts, that cause the browser to lag.

The dashboard files are served through cloudflare, especially to avoid this kind of issues. The first time you access the dashboard, your browser may need some more time to download everything, but refreshes should be instant. In my tests, I see 1-2 seconds to end up with charts on my browser.

If this is not the case for you, we need to identify the issue and fix it. Netdata dashboards should not need more than 2 seconds to show charts.

### Multi-node, Multi-instance charts

> In addition that Node also acts as a replication receiver of several other nodes.
> What I absolutely do not need as a unified graph where all the interfaces are summed! Changing that to 'Average' 'Min' or 'Max' did not change a thing in the graph.
> Trying to select a singular interface showed even worse behavior. Most of my Nodes have 'ens3' as their primary network interface. Those are all merged together into a single one and I was not able to split them apart in the new trashboard.
Another huge issue is that graphs that are supposed to be viewed side-by-side can't be viewed in such a way at all anymore. In this case: fping! I'm monitoring the ping times between my nodes and a few public servers. Again: What I don't need AT ALL is the sum of the latencies. I also don't need an average, which doesn't even work as before!

I understand your frustration. I promise we will give you back per instance charts asap. The release of the new UI is planned for Wed. Immediately after we will work on drilling down instances into individual charts.

Keep in mind that the new charts are multi-node and multi-instance. And we have done a lot of work to help using them, quite easily.

You can filter nodes globally from the top right of your screen:

![image](https://github.com/netdata/netdata/assets/2662304/b3005786-1930-4009-ad57-8273890e1b96)

You can also filter nodes from within each chart:

![image](https://github.com/netdata/netdata/assets/2662304/dcd6846a-3ef8-4577-8734-b866000d471f)

The instances drop-down menu (called `interfaces` in network interfaces), also shows all the network interfaces both by their name and the host they belong. So, even if all your network interfaces are called `ens3` you can still see where they belong to:

![image](https://github.com/netdata/netdata/assets/2662304/39dc7943-d1d3-4b3c-ac09-55d9fb8cec58)

### Not working selectors

> Using the "local copy" of the trashboard fixes the not working selectors for the graphs at least. But the really doesn't make the new trashboard any more usable.

Help me understand this. There are selectors that are not working for you? The "local copy" and the live versions of the dashboard are currently the exactly the same.

Can you share a small video or something to help us identify the issue?

---

Overall, the old dashboard had severe limitations and was not working for a lot of users. The new dashboard has a tremendous amount of work on it.

Please help us spot the issues and solve them. I understand your frustration and I promise we will do our best to solve everything immediately. We count on your satisfaction, so we truly and honestly will do everything in our power to make this work.

**Reply by Pingger** [reply]
> From your message I understand that the main problem is that you need to have the network interfaces expanded. So, instead of having multiple interfaces in one chart, you need one network interface per chart. Correct me if I got it wrong.

Yes, but it doesn't only affect network interfaces. Additionally: Apache, btrfs, Disks, Mount Points, Users, ssh, filecheck (defeats the purpose if can't check which file has the issue), Minecraft Servers, TeamSpeak Servers, fping, Docker and **Linux Containers** (LXC/LXD), are also among the critically essential things that must not EVER be merged into single graph.

In essence. Most things must not be merged into singular graphs, not even on a singular node. The few things, that are somewhat ok for now are CPU, Anomaly Detection and Netdata Monitoring. For everything else I would MUCH prefer split up graphs

As a small example: This:
![image](https://github.com/netdata/netdata/assets/4761298/c80f35a9-ae80-4613-80e9-76cf569c8ee5)
got merged down to this:
![image](https://github.com/netdata/netdata/assets/4761298/8ec219a3-a1c8-4d98-89fd-bde2804d013d)


> The dashboard files are served through cloudflare, especially to avoid this kind of issues. The first time you access the dashboard, your browser may need some more time to download everything, but refreshes should be instant. In my tests, I see 1-2 seconds to end up with charts on my browser.

That would explain that. Cloudflare services are supposed to be completely blocked. Apparently we missed a few servers. (Among the reasons being their untenable handling of private data and their Privacy Policy)

> You can also filter nodes from within each chart:
Doesn't really help get a good overview, when I have to do several clicks to get the next graph. I have ~20 Graphs that I check daily that got merged down to 2 Graphs. And an additional 40 Graphs, that I usually check once a week, which got merged down to another 3 Graphs. 
So in essence, I have to do ~60 additional clicks for the daily graphs and ~120 more for the weekly ones. Clicks I didn't have to do before, but could just 'scroll past'.

> 
> Help me understand this. There are selectors that are not working for you? The "local copy" and the live versions of the dashboard are currently the exactly the same.
> 
> Can you share a small video or something to help us identify the issue?

I'm quite sure this might be caused by the cloudflare block. Explaining why it worked in the local copy but not "default" cloud copy (most of the 'cloud' part was apparently still blocked).



Something else, that might be influencing the bad performance of the new dashboard:
My Netdata-database holds stats for the last 5-6 Months. (with some holes, where the updater broke, or hardware maintenance was required, but mainly broken updater failing to properly update or being to stupid to restart netdata after the update)
![image](https://github.com/netdata/netdata/assets/4761298/d66e07d8-8fd8-4983-a118-70a254c8f742)



Unmarked answer, as it doesn't answer the question and even after pointing that out, no further response was offered!

**Reply by Pingger** [reply]
Something I also just noticed: v2 misrepesents CPU-Usage of Containers. It for some reason divides it by 10!
So a container using 128% cpu (aka 1.28 Cores), now uses 12.8% according to the new dashboard?! 
And maximum CPU utilisation of containers is even weirder.
That same container apparently uses 50% CPU maximum. How it can do that I have no idea. 
That specific container may use 2 Cores, and generally caps one core and holds the other one at 20-30%.
So the maximum would have to be either: 130%, when doing per Core. ~65% when doing "of maximum in limit". or ~20% when doing "of maximum the host system has" (8 Cores). 
So I can't even remotely understand the "50%" (or 51.4%) displayed.
![image](https://github.com/netdata/netdata/assets/4761298/3db6c821-0411-4aa3-9c13-d3a4deb9b829)

**Reply by Cisco30** [reply]
Hi, I'm lost!!! what happened ? can we recover the old interface? and if so how to do it, thank you

**Reply by andrewm4894** [reply]
@Cisco30 if you go to /v1 you can get to old dashboard - like this: https://sanfrancisco.my-netdata.io/v1

Or really really old one is here

https://sanfrancisco.my-netdata.io/v0
Or
https://sanfrancisco.my-netdata.io/old
(Same thing)

**Reply by Cisco30** [reply]
@andrewm4894  thank you very much for your solution

**Comment by aldem** [comment]
The original question (I second it) remains unanswered - how to disable the new dashboard or use v1 as the default at least?

While the new dashboard has a lot of really excellent features it is also very confusing and makes more difficult to find relevant data (while some data is even wrong on the first sight), there should be a way to use v1 or even v0 by default - for some tasks (and where amount of data is not so enormous) it would be really nice.

The old dashboards could have a link to the new one, or v2 could contain easily accessible links to v1/v0 - changing URL manually is a pain. And no, not everyone is using cloud, registry or even bookmarks.

This is really annoying when with every new major release of dashboard something changing in a way that ruins your routines - imagine that you get an upgrade for your car's engine, it is twice as powerful now but the clutch is on the right and the gas is on the left - sure you will get used to the new layout, but in the meantime (long enough) it will make you cry at least.

**Comment by andrewm4894** [comment]
fyi - i made a feature request here to start gathering feedback for improvements around how to "expand" or "split" charts somewhat similar to how used to be https://github.com/netdata/netdata-cloud/issues/894
